src/planner/ddl_plan.go

```diff
@@ -137,7 +137,16 @@ func (p *DDLPlan) Build() error {
 				rawQuery := strings.Replace(p.RawQuery, "`", "", 2)
 				// \b: https://www.regular-expressions.info/wordboundaries.html
 				re, _ := regexp.Compile(fmt.Sprintf(`\b(%s)\b`, table))
-				query = re.ReplaceAllString(rawQuery, segTable)
+				// avoid the name of the column is the same as the table name, eg, issues/438
+				var count = 0
+				var occurrence = 1
+				query = re.ReplaceAllStringFunc(rawQuery, func(m string) string {
+					count = count + 1;
+					if (count == occurrence) {
+						return segTable
+					}
+					return m
+				})
 			} else {
 				segTable = fmt.Sprintf("`%s`.`%s`", database, segTable)
```

Comment:
`else` statement has `re.ReplaceAllString` , I think we should handle it too.
And it's better move the Replace to out:
```
			if node.Table.Qualifier.IsEmpty() {
				segTable = fmt.Sprintf("`%s`.`%s`", database, segTable)
				rawQuery := strings.Replace(p.RawQuery, "`", "", 2)

			} else {
				segTable = fmt.Sprintf("`%s`.`%s`", database, segTable)
				newTable := fmt.Sprintf("%s.%s", database, table)
				rawQuery := strings.Replace(p.RawQuery, "`", "", 4)
			}

        do replace something... ...
```

Reply:
the else statement has re.ReplaceAllString, but the err case won't happen.

Reply:
Have changed it.